deltachat-ffi/deltachat.h

```diff
@@ -5746,6 +5760,32 @@ void dc_event_unref(dc_event_t* event);
 /// Used in message summary text for notifications and chatlist.
 #define DC_STR_FORWARDED                  97
 
+/// "Your mailbox on your email account is running full! [...]"
+///
+/// Send as device message to the user when their mailbox is nearly full
+///
+/// The message is longer and also contains suggestions for possible solutions.
+#define DC_STR_QUOTA_MAILBOX_NEARLY_FULL  98
+
+/// "Your email server does not support the quota extension"
+///
+/// Send as device message to the user when their email provider does not support the quota extention.
+#define DC_STR_QUOTA_NOT_SUPPORTED        99
+
+/// "Messages"
+///
+/// Used in the quota report message.
+///
+/// Example usage: Messages: 3912/6000
+#define DC_STR_QUOTA_MESSAGES            100
+
+/// "Storage"
+///
+/// Used in the quota report message.
+///
+/// Example usage: Storage: 169.33 MiB/1000 MiB
+#define DC_STR_QUOTA_STORAGE             101
```

Comment:
I'm not an expert in translation (we can ask Björn when he is back), but I think that this should use templates, like for example ContactSetupChanged does ("Changed setup for %1$s"):

For right-to-left languages, we can't just put the value to the right. If we use templates, translators can do "%1$ :xyz" to to put the value to the left.

Reply:
it is probably fine as the string is used in `DC_STR_QUOTA_USAGE` - where you can adapt the order or right to left or so.

but if there are only two quota types ($type_name in DC_STR_QUOTA_USAGE), maybe it is simpler to combine DC_STR_QUOTA_USAGE/DC_STR_QUOTA_STORAGE/DC_STR_QUOTA_MESSAGES to only two, more simpler string as:

```
Storage: %1$s/%2$s
Messages: %1$s/%2$s
```

that would also be easier to explain, a otherwise might be replaced by slightly different things.

Reply:
> but if there are only two quota types

there are/can be more than those two types: https://www.rfc-editor.org/rfc/rfc2087.html#section-3

Reply:
units should not be added by translators

Reply:
I changed it so there are now 3 strings all with the 2 variables for $usage and $limit:

Storage: `Storage: %1$s/%2$s`
Messages: `Messages: %1$s/%2$s`
other: `%1$s: %2$s/%3$s`

Reply:
> units should not be added by translators

my point was that one has to explain that %1$s is sometimes replaced by a number and sometimes by number+unit. but anyway, that is fixed now :)